mlkem/native/api.h

```diff
@@ -249,7 +319,15 @@ static INLINE void poly_frombytes_native(int16_t a[MLKEM_N],
  * len).
  **************************************************/
 static INLINE int rej_uniform_native(int16_t *r, unsigned int len,
-                                     const uint8_t *buf, unsigned int buflen);
+                                     const uint8_t *buf, unsigned int buflen)
+__contract__(
+  requires(len <= 4096 && buflen <= 4096 && buflen % 3 == 0)
```

Comment:
Should the 4096 here be UINT12_LIMIT?

Reply:
No, it's a fairly arbitrary bound that is semantically unrelated to the UINT12_LIMIT